.idea/watcherTasks.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like an IDE config file. Usually want to add these to your global git ignore settings so they don't end up in commits. I can get a reference on this to you if you haven't done it before.